scio-core/src/main/scala/com/spotify/scio/testing/TestDataManager.scala

```diff
@@ -112,6 +113,8 @@ case class BigQueryIO(tableSpecOrQuery: String) extends TestIO[TableRow](tableSp
 case class DatastoreIO(projectId: String, query: Query = null, namespace: String = null)
   extends TestIO[Entity](s"$projectId\t$query\t$namespace")
 
+case class ProtobufIO[T](path: String)(implicit ev: T <:< Message) extends TestIO(path)
```

Comment:
But that already has `ev: T <:< Message` so this shouldn't need it. We don't have the constraint for `AvroIO` either.
Assuming user uses the right `T` in `.output[T](ProtobufIO(...))` which is also constrained by the assertion inside.
So no need to go nuts here.